<launch>
    <include file="$(find jackal_bsc/launch/bsc.launch"/>
    <rosparam command="load" file="$(find jackal_bsc)/launch/exp_params.yaml" ns="jackal_bsc"/>
    <rosparam command="load" file="$(find jackal_bsc)/launch/delay_params.yaml" ns="jackal_bsc"/>
    <node name="delay_node" package="jackal_bsc" type="delay_node.py"/>
</launch>